README.md

```diff
@@ -1,55 +1,76 @@
   * e.g. "add @dtoki, and @alireza to artris"
-  * When there is a confilict, the hacker will not be added to the new stand-up and a warning will be displayed
-* `/lj remove [me or a list @hacker] from [name]`
-  Remove hackers from an existing stand-up
+  * When there is a conflict, the hacker will not be added to the new stand-up and a warning will be displayed
+
+* #### Remove hackers from an existing stand-up
+    `/lj remove [me or a list of @hacker] from [name]`
   * e.g. "remove me from artris"
-* `/lj move [name] to [when]`
-  Reschedule a stand-up
-  * e.g. "move artris to 7am"
-* `/lj terminate [name]`
-  Terminate a stand-up
-  * e.g. "terminate artris"
-* `lj halt [name]`
-  Nofity the bot when the team is on a break
+  * An error message will be displayed if the hacker does not belong to the specified stand-up
+
+* #### Reschedule a stand-up
+    `/lj move [name] to [when]`
+  * e.g. "move artris to everyday at 7:00 am"
+  * An warning will be displayed if the new schedule results in conflicts among participant hacker schedules
+
+* #### Temporarily halt a stand-up
+    `lj halt [name]`
   * e.g. "halt artris"
-* `lj resume [name]`
-  Nofity the bot when you want to restart the stand-ups
+
+* #### Resume a stand-up
+    `lj resume [name]`
   * e.g. "resume artris"
-* `/lj status`
-  Display info about the active stand-ups and member participation
-* `/lj [I or @hacker] will skip [name] [time range]`
-  Notify the bot before taking a break
-  * At least 8h before the stand-up
+
+* #### Terminate a stand-up
+    `/lj terminate [name]`
+  * e.g. "terminate artris"
+
+
+* #### Notify an individual's timed break for a stand-up
+    `/lj [I or @hacker] will skip [name] [time range]`
+
+  * Allowed at least 8h before the stand-up
   * e.g. "I will skip artris for 2 weeks"
-* `/lj start notifying [username] for [name]`
-  Notify the bot that the hacker will start attending a stand-up again after a 'stop' or 'skip' command has been sent
-  * e.g. "start notifying @grace for artris"
-* `/lj stop notifying [username] for [name]`
-  The hacker is on a break and will not be counted as part of a stand-up until the 'start' command is sent
+  * An error message will be displayed if the hacker is not part of the standup
 
+* #### Opt out from a stand-up
+    `/lj stop notifying [me or list of @hacker] for [name]`
+  * The hacker is on a break and will not be counted as part of a stand-up until a corresponding 'start' command is sent
   * e.g. "stop notifying @grace for artris"
+  * An error message will be displayed if the hacker is not part of the standup
+
+* #### End break and notify standup attendance
+    `/lj start notifying [me or list of@hacker] for [name]`
+  * e.g. "start notifying @grace for artris"
+  * An error message will be displayed if the hacker is not part of the standup
+
+* #### Display info on all active stand-ups and member participation
+    * `/lj status`
+
 
-## how does it work?
+## How does it work?
```

Comment:
Shouldn't this be a the top of the documentation before the user uses the documentation?

Reply:
No I think it's fine here because we also add the database schema at the bottom. You could add more about the database scheme, the reason we have logs etc ...